static-site/assets/swagger-overrides.scss

```diff
@@ -0,0 +1,54 @@
+---
+--- 
+
+// all of this is needed to prevent the base theme CSS from interfering with Swagger's CSS
+
+#swagger-ui {
```

Comment:
This is unfortunate, but since Jekyll doesn't use any kind of css scoping, we need to deal with clashes between Swagger and the default css provided by the theme.